server/src/main/java/org/jboss/as/server/controller/resources/ServerRootResourceDefinition.java

```diff
@@ -259,6 +259,8 @@ public void registerOperations(ManagementResourceRegistration resourceRegistrati
 
         // Other root resource operations
         resourceRegistration.registerOperationHandler(CompositeOperationHandler.DEFINITION, CompositeOperationHandler.INSTANCE, false);
+        //resourceRegistration.registerOperationHandler(MapReduceOperationHandler.DEFINITION, MapReduceOperationHandler.INSTANCE, false);
```

Comment:
Please remove. The original ordering of the imports was the correct one as well.